src/virtualenv/activation/python/activate_this.py

```diff
@@ -20,18 +20,18 @@
     raise AssertionError(msg) from exc
 
 bin_dir = os.path.dirname(abs_file)
-base = bin_dir[: -len("__BIN_NAME__") - 1]  # strip away the bin part from the __file__, plus the path separator
+base = bin_dir[: -len(__BIN_NAME__) - 1]  # strip away the bin part from the __file__, plus the path separator
 
 # prepend bin to PATH (this file is inside the bin directory)
 os.environ["PATH"] = os.pathsep.join([bin_dir, *os.environ.get("PATH", "").split(os.pathsep)])
 os.environ["VIRTUAL_ENV"] = base  # virtual env is right above bin directory
-os.environ["VIRTUAL_ENV_PROMPT"] = "__VIRTUAL_PROMPT__" or os.path.basename(base)  # noqa: SIM222
+os.environ["VIRTUAL_ENV_PROMPT"] = __VIRTUAL_PROMPT__ or os.path.basename(base)
 
 # add the virtual environments libraries to the host python import mechanism
 prev_length = len(sys.path)
-for lib in "__LIB_FOLDERS__".split(os.pathsep):
+for lib in __LIB_FOLDERS__.split(os.pathsep):
     path = os.path.realpath(os.path.join(bin_dir, lib))
-    site.addsitedir(path.decode("utf-8") if "__DECODE_PATH__" else path)
+    site.addsitedir(path.decode("utf-8") if __DECODE_PATH__ else path)
```

Comment:
I don't think you meant to remove quotes in `.py` files, e.g. `"__VIRTUAL_PROMPT__"` -> `__VIRTUAL_PROMPT__`. These simply become unassigned Python identifiers, and an error will be thrown whenever `src/virtualenv/activation/python/activate_this.py` is executed (imported).

Reply:
Or more likely I don't understand what and where sets them and _why the quotes were present before this change_. With quotes, the code does not make sense to me either.

Reply:
Those values are replaced with python representation here https://github.com/pypa/virtualenv/blob/main/src/virtualenv/activation/python/__init__.py#L21-L25.

Reply:
So in this case, the quotes must be there...

Reply:
Nope, because the replacer inserts it via quotes https://github.com/pypa/virtualenv/blob/main/src/virtualenv/activation/via_template.py#L76

Reply:
> This is a template file.

Oh, I got it know. The _file_ is not meant to be interpreted by Python before pre-processing. Thanks for clarifying this for me!